Freshness pass: Introduction to .NET (#52136)
dotnet/docs: docs/core/introduction.md

---
title: Introduction to .NET
description: Learn about .NET. .NET is a free, open-source development platform for building many kinds of apps.
ms.date: 03/10/2026
ms.custom: "updateeachrelease"
ai-usage: ai-assisted
---
# Introduction to .NET

.NET is a free, cross-platform, [open-source developer platform](https://github.com/dotnet/core) for building [many kinds of applications](apps.md). It can run programs written in [multiple languages](../fundamentals/languages.md), with [C#](../csharp/index.yml) being the most popular. It relies on a [high-performance](https://devblogs.microsoft.com/dotnet/category/performance/) runtime used in production by many [high-scale apps](https://devblogs.microsoft.com/dotnet/category/developer-stories/).

To learn how to [download .NET](https://dotnet.microsoft.com/download/) and start writing your first app, see [Getting started](./get-started.md).

> The .NET platform has been designed to deliver productivity, performance, security, and reliability. It provides automatic memory management via a [garbage collector (GC)](../standard/automatic-memory-management.md). It's type-safe and memory-safe, because it uses a GC and strict language compilers. It offers [concurrency](../csharp/asynchronous-programming/index.md) via `async`/`await` and `Task` primitives. It includes a large set of libraries that have broad functionality and have been optimized for performance on multiple operating systems and chip architectures.

Reviewer: ```suggestion
The .NET platform is designed to deliver productivity, performance, security, and reliability. It provides automatic memory management via a [garbage collector (GC)](../standard/automatic-memory-management.md). It's type-safe and memory-safe, because it uses a GC and strict language compilers. It offers [concurrency](../csharp/asynchronous-programming/index.md) via `async`/`await` and `Task` primitives. It includes a large set of libraries that have broad functionality and that are optimized for performance on multiple operating systems and chip architectures.
```
